### ISSUE TITLE
Feature request: public badges for private jobs

### ISSUE THREAD

**Issue by gabro** [concourse/concourse#996]
# Feature Request

## What challenge are you facing?

GitHub badges for private jobs don't show up in GitHub READMEs.
Here's how I see them:

![image](https://cloud.githubusercontent.com/assets/691940/24606324/f7bddd40-186c-11e7-980a-f046a77f7b51.png)

## A Modest Proposal

It'd be lovely to have an option to make badges for a job available publicly, regardless of the job being private.

**Comment by chendrix** [comment]
Is [exposing your pipeline](http://concourse.ci/fly-expose-pipeline.html) not an available option in this use case?

**Comment by gabro** [comment]
Not really, the pipeline is private and I want it to stay like that, and the badges would need to appear on a private GitHub repo

--
gabriele
~ buildo

**Comment by wieslaw-gat** [comment]
I have the exact same use case as @gabro - would love to see this implemented.

**Comment by simonjohansson** [comment]
+1

We have many many teams in concourse and making all pipelines public just to be able to have badges is quite a thorn in the side. :)

**Comment by BenElgar** [comment]
+1 (for the sake of the bot)

I would find it useful and, judging by the duplicate issue opened 11 days ago, I'm not completely alone.

**Comment by makslevental** [comment]
+1

**Comment by vito** [comment]
Ended up reverting the PR to get 6.0 out, so reopening!

**Comment by raymondberg** [comment]
Highlighting [a good workaround](https://github.com/concourse/concourse/issues/610#issuecomment-372461674) from @emcniece on #618: 

> I made a deploy job that pushes the badges to an S3 bucket as a workaround 😅

Also +1 this feature

**Comment by dwrolvink** [comment]
Has this been implemented in the mean time?

I just spent too much time creating a script to combine the status of all our environments on one page, only to find out that I still need to log in to every single one to see the badges :sweat_smile:

**Comment by taylorsilva** [comment]
Discussion here is relevant if anyone wants to pick this up in the future: https://github.com/concourse/concourse/pull/5308#issuecomment-600283564

# Issue Bankruptcy

This issue is being closed as part of 2025's **Issue Bankruptcy** declaration. [See here for details](https://blog.concourse-ci.org/posts/2025-01-06-concourse-2024-review/)

If you think this issue still has merit then feel free to open a new issue and reference this one.
